chsdi/templates/htmlpopup/schienennetz_point.mako

```diff
@@ -0,0 +1,31 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+<%
+
+  abreviation = c['attributes']['respdonneesabreviation']
+
+%>
+
+    <tr><td class="cell-left">${_('ch.bav.schienennetz.xtf_id')}</td>                                            
+	<td>${c['featureId'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.bav.schienennetz.numero')}</td>
+    <td>${c['attributes']['numero'] or '-'}</td></tr>    
+    <tr><td class="cell-left">${_('ch.bav.schienennetz.nom_point')}</td>
+    <td>${c['attributes']['nom_point'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.bav.schienennetz.abreviation')}</td>
+    <td>${c['attributes']['abreviation'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.bav.schienennetz.respdonneesabreviation')}</td>
+% if abreviation == 'SBBCFFFFS':
```

Comment:
Such things could be solved during data import, no? Or are there other interfaces that need `SBBCFFFFS` as a string?